### ISSUE TITLE
Linkgrabber.add_links not working

### ISSUE THREAD

**Issue by MJD96** [mmarquezs/My.Jdownloader-API-Python-Library#36]
I used this code, but the only parameter that seems to work is "links". Because when I open Jdownlader, the link is added but  everything is set by default.

```
lincs=myjdapi.myjdapi.Linkgrabber(device) #objeto del Linkgrabber

url='https://www.youtube.com/watch?v=d6DP_u6ogqc' 
youtuber = 'Ciudadano Z'
fecha = time.ctime()
lincs.add_links([{
                      "autostart": False,
                      "links": url,
                      "packageName": 'tu vieja',
                      "extractPassword": None,
                      "priority": "DEFAULT",
                      "downloadPassword": None,
                      "destinationFolder": r'C:\Users\MARIANO\\' ,#Downloads\Jdownloader\\'+ youtuber + '\\' + fecha,
                      "overwritePackagizerRules": False
                  }])
```

**Comment by mmarquezs** [comment]
You aren't the first to be confused by this. The thing is that JDownloader has a "Packagizer". This "Packagizer" automatically sets up the destination path, creates subfolder, automatically names the folder/package and other things. 

So even if you set up those fields manually if the packagizer is active it will override them once the link is added to the linkgrabber. 
If you want to avoid that you either modify the Packagizer settings or you use the flag to overwrite the packagizer rules.

To be honest if you want to modify the Packagizer settings there are some issues, first I am not sure the api supports it and if it does, my api doesn't have specific functions for it so it probably requires to use the GUI to change the its settings. Second, some things I don't think they can be disabled, for example although I managed to disable the rule that overwrites the destination folder I couldn't find a way to disable the package name overwrite from the Packagizer.  In any case the settings are in Settings -> Packagizer (or in Spanish Opciones -> Empaquetador) )

It's much easier to use the flag, you only have to set the "**overwritePackagizerRules**" as True and that's it. 
I have checked, so far it works as intended. Feel free to close the issue if that's all.
